src/cql/r4/Factors_to_Consider_in_Managing_Chronic_Pain_FHIRv400.cql

```diff
@@ -846,6 +846,29 @@ define NonOpioidMedicationRequests:
     100 years //changed from 2 to 100
   ))
 
+define PDMPMedicationRequests:
+  [MedicationRequest: "Opioid Pain Medications"] M 
+  where M.identifier is not null and (
+    M.identifier[0].system ~ 'https://github.com/uwcirg/script-fhir-facade/'
```

Comment:
@achen2401 we're only passing the system at this time - no need to check for `value`